### DISCUSSION TITLE
Add Gitea & Forgejo Instance & Username Field in users bio

### DISCUSSION THREAD

**Discussion by trymeouteh** [WeblateOrg/weblate#11261] [Ideas]
You can add a Github username in your username bio. Please add fields to add your Gitea and Forgejo instance and username in your weblate account bio.

**Comment by nijel** [comment]
It doesn't make sense to ask for username in that case as that needs full URL to include server as well. And there is already a field for that in the profile: https://github.com/WeblateOrg/weblate/blob/21ff9e8c76a5b87819235f43d714c3666ba9e4cb/weblate/accounts/models.py#L564-L570
